artifactory/commands/terraform/terraformpublish.go

```diff
@@ -54,11 +54,17 @@ func (tpc *TerraformPublishCommand) SetArgs(terraformArg []string) *TerraformPub
 	return tpc
 }
 
-func (tpc *TerraformPublishCommand) SetServerDetails(serverDetails *config.ServerDetails) *TerraformPublishCommand {
+func (tpc *TerraformPublishCommand) setServerDetails(serverDetails *config.ServerDetails) *TerraformPublishCommand {
 	tpc.serverDetails = serverDetails
 	return tpc
 }
 
+func (tpc *TerraformPublishCommand) setRepoConfig(conf *utils.RepositoryConfig) *TerraformPublishCommand {
+	serverDetails, _ := conf.ServerDetails()
```

Comment:
It looks like the signature of the `conf.ServerDetails()` function should be changed to not return an error, beucase the value returned for the error is always nil.

Reply:
This is part of the implementation of the command interface